frontend/src/index.css

```diff
@@ -429,6 +475,40 @@ Other button classes are defined further down together with other classes for th
   margin: 0 0 0 3.75rem;
 }
 
+.advanced-search-container {
+  min-height: 270px;
+}
+
+.autocomplete-container {
+  margin-left: -3.2em;
+  margin-top: 0.6em;
+  list-style-type: none;
+  max-height: 100px;
+  overflow-y: auto;
+  z-index: 1;
+}
+
+.autocomplete-item {
+  padding: 0 24px;
```

Comment:
if you change the padding like this, there won't be a gap to the right between the scrollbar and the autocomplete buttons:
```suggestion
  padding: 0 0 0 24px;
```